tests/benchdnn/softmax/softmax.cpp

```diff
@@ -240,10 +240,12 @@ void setup_cmp(compare::compare_t &cmp, const prb_t *prb, data_kind_t kind,
     // minor accuracy drops observed compared to f32
     const float trh = trh_f32;
 #else
-    const bool is_strict_acc
-            = prb->attr.acc_mode == dnnl_accumulation_mode_strict;
+    const bool is_strict_acc = !is_nvidia_gpu()
```

Comment:
why not add DNNL_SYCL_CUDA in the `#if` condition above ?

Reply:
Changed the way the threshold is added to use DNNL_SYCL_CUDA